src/main/java/seedu/address/model/module/UniqueModuleList.java

```diff
@@ -83,9 +83,9 @@ public Module getModule(Module module) {
     }
 
     /**
-    * Gets module with given module code.
-    * Returns null if no such module exists.
-    */
+     * Gets module with given module code.
+     * Returns null if no such module exists.
+     */
```

Comment:
Ehh hope this works but @aakanksha-rai just check that IntelliJ is indenting your comments correctly?